File: app/src/main/java/com/pisces/android/wuha/mine/ShareActivity.java
package com.pisces.android.wuha.mine;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.pisces.android.framworkerlibrary.core.LBaseActivity;
import com.pisces.android.wuha.R;

/**
 * Created by Chris Li on 2017/9/1.
 * 分享app界面
 */

public class ShareActivity extends LBaseActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_share);
        setToolbarTitle("分享app");
    }
}

File: app/src/main/java/com/pisces/android/wuha/mine/ShareActivity.kt
package com.pisces.android.wuha.mine

import android.os.Bundle

import com.pisces.android.wuha.R
import com.pisces.android.wuha.base.LBaseActivity

/**
 * Created by Chris Li on 2017/9/1.
 * 分享app界面
 */

class ShareActivity : LBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_share)
        setToolbarTitle("分享app")
    }
}
